packages/jaeger-ui/src/components/SearchTracePage/index.jsx

```diff
@@ -100,25 +98,31 @@ export class SearchTracePageImpl extends Component {
     const hasTraceResults = traceResults && traceResults.length > 0;
     const showErrors = errors && !loadingTraces;
     const showLogo = isHomepage && !hasTraceResults && !loadingTraces && !errors;
+    const tabItems = [];
+    if (!loadingServices && services) {
+      tabItems.push({ label: 'Search', key: 'searchForm', children: <SearchForm services={services} /> });
+    } else {
+      tabItems.push({ label: 'Search', key: 'searchForm', children: <LoadingIndicator /> });
+    }
+    if (!disableFileUploadControl) {
+      tabItems.push({
+        label: 'Upload',
+        key: 'fileLoader',
+        children: (
+          <FileLoader
+            loadJsonTraces={fileList => {
+              loadJsonTraces(fileList);
```

Comment:
why is the lambda used here instead of simply referencing the function name? I think it would remove the code coverage warning

Reply:
Right actually. I just copied this piece of code from the previous one.
Removed lambda function from this part, though, the coverage is because this part has no test for it, previously too.